inc/class-wpseo-utils.php

```diff
@@ -61,7 +61,15 @@ public static function is_apache() {
 
 		$software = sanitize_text_field( wp_unslash( $_SERVER['SERVER_SOFTWARE'] ) );
 
-		return stripos( $software, 'apache' ) !== false;
+		if ( stripos( $software, 'apache' ) !== false ) {
+			$abool = true;
+		}
+		elseif ( stripos( $software, 'litespeed' ) !== false ) {
+			$abool = true;
+		}
+		else $abool = false;
+
+		return $abool;
```

Comment:
It can be done without if/elseif/else...:
```suggestion
		return stripos( $software, 'apache' ) !== false || stripos( $software, 'litespeed' ) !== false;
```

Reply:
That is much simpler isn't it? Thanks!